File: Codefest Final/round 1/problem3.py
a = "Aliquam erat volutpat. Etiam eu ligula ligula."
b = "Lorem ipsum dolor sit amet, consectetur adipiscing elit."

arr1 = {}
arr2 = {}

for i in a:
    arr1[i] += 1

for i in b:
    arr2[i] += 2

for x, y in arr1:
    if arr1[x]==arr2[x]:
        print(x)

File: Codefest Final/round 1/problem3.cpp
#include<bits/stdc++.h>
using namespace std;

int main(){
    string a = "Aliquam erat volutpat. Etiam eu ligula ligula.";
    string b = "Lorem ipsum dolor sit amet, consectetur adipiscing elit.";

    map<char, int> mp1;
    map<char, int> mp2;

    for(int i=0; i<a.length(); i++){
        mp1[a[i]]++;
    }
    for(int i=0; i<a.length(); i++)
        mp2[b[i]]++;

    for(auto it: mp1){
        if(mp1[it.first]<=mp2[it.first]){
            cout << it.first << " ";
        }
    }
}